src/cameras/PerspectiveCamera.js

```diff
@@ -135,6 +151,18 @@ THREE.PerspectiveCamera.prototype.copy = function ( source ) {
 	this.near = source.near;
 	this.far = source.far;
 
+	this.zoom = source.zoom;
+	this.focalLength = source.focalLength;
+
+	this.skewSlopeX = source.skewSlopeX;
+	this.skewSlopeY = source.skewSlopeY;
+
+	if ( source.view !== null && source.view !== undefined ) {
```

Comment:
> I think only checking for `null` should be enough?

It's called by the loader and since it didn't get serialized before might be missing, thus `=== undefined`.

> > I'm unclear why not using `undefined` isn't prefered?

Because there's a conceptual difference between a missing property and an existing property which currently doesn't bind to an object. Also it's lengthy.

> Because that changes V8 hidden object.

So does defining a property with a value of `undefined`, actually (`for...in` must find it, for example). Also, hidden types are not a speciality of V8 - they exist in pretty much all JS engines used by browsers, and (as long as the order in which properties are added is the same) the transitions extending objects are cached in all of them. Extending an object still needs reallocation, because the extended type needs more space to store its extra properties.